src/browser/keyboard.js

```diff
@@ -75,6 +75,8 @@ define((require, exports, module) => {
 
         if (binding) {
           binding(...args);
+          event.preventDefault();
+          event.stopPropagation();
```

Comment:
> That's a bit violent. Can this be handled in the callback?

At the moment non of our keybindings code needs events to be propagated, how about we keep it and change when / if we need it ?